tests/test_cli.py

```diff
@@ -5,10 +5,18 @@
 import pytest
 import click.testing
 import archspec.cli
+from archspec import __version__
```

Comment:
Minor OCD. Can we maintain the namespace, i.e.
```suggestion
import archspec
```
and use `archspec.__version__` below?

Reply:
fixed in 2acc291